backend/src/app.module.ts

```diff
@@ -1,14 +1,35 @@
+import { UserModule } from './user/user.module'
 import { Module } from '@nestjs/common'
 import { AppController } from './app.controller'
 import { AppService } from './app.service'
 import { HttpModule } from '@nestjs/axios'
 import { UserProfileController } from './userProfile/UserProfile.controller'
 import { UserProfileService } from './userProfile/UserProfile.service'
+import { TypeOrmModule } from '@nestjs/typeorm'
+import { ConfigModule } from '@nestjs/config'
+import path from 'path'
```

Comment:
`import path from 'path'` does not work since the module has undefined properties, that is why according to documentation it should be `import * as path from 'path'`